lib/shopt.sh

```diff
@@ -27,7 +27,15 @@ shopt -s nocaseglob;
 ## SMARTER TAB-COMPLETION (Readline bindings) ##
 
 # Perform file completion in a case insensitive fashion
```

Comment:
Please update the comment

Reply:
@saul365 can you address this too?
The comment in the code is now inaccurate and needs to be amended.
Thanks!

Reply:
This is supposed to be addressed in #399 as I see the phrase

> address the remaining open suggestion in it. 

in https://github.com/ohmybash/oh-my-bash/pull/399#issue-1586451593, though I don't see the update to this line.